### PR TITLE
Fix to handle fancybox customer creation in Add an Order page

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix to handle fancybox customer creation in Add an Order page
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/13013
| How to test?  | See fixed ticket behavior

This is a port of legacy behavior in https://github.com/PrestaShop/PrestaShop/issues/13013#issuecomment-476002724 . I considered modifying the system but, as explained in comment below, this would have a great impact. It's better to wait for Add an order page to be migrated to fully fix this.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/13043)

### CODE REVIEW

**Comment by matks** [comment]
The right way to do it would be to close the fancybox when we get an HTTP 200 from the form submission. But the form is submitted in very generic file https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/default/template/form_submit_ajax.tpl#L51 😐

File: src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php [hunk 1]
```diff
@@ -130,6 +130,17 @@ public function createAction(Request $request)
             if ($customerId = $result->getIdentifiableObjectId()) {
                 $this->addFlash('success', $this->trans('Successful creation.', 'Admin.Notifications.Success'));
 
+                $formWasSubmittedAsAjax = ($request->query->has('submitFormAjax'));
+                if ($formWasSubmittedAsAjax) {
```
Comment (PierreRambaud) [inline src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php]: The parameter name talks itself
```suggestion
                if ($request->query->has('submitFormAjax')) {
```

**Review by PierreRambaud — DISMISSED** [review]

**Comment by marionf** [comment]
@matks 

If I create a new order, click on add a new customer and click on Cancel, I still have the PrestaShop's inception

capture d'écran_1232

**Comment by matks** [comment]
This code will be disabled/removed in PrestaShop 1.7.7 as we aim to shop a migrated version of "Add an Order" page and we'll rework the fancybox being used in this page at this time (allowing us to deal with this in a better way).

**Review by jolelievre — APPROVED** [review]

**Comment by marionf** [comment]
I just created an issue if we can fix it during code freeze #13085

**Comment by PierreRambaud** [comment]
Thanks @matks

**Pull request merged**